EasyLearn/app/assets/stylesheets/application.css

```diff
@@ -13,3 +13,6 @@
  *= require_tree .
  *= require_self
  */
+body{
+    background-color: rgba(33, 33, 41, 1)!important;
```

Comment:
I recommend you not to use `!important`, since it is considered a bad practice and can cause problems in the development of your styles.

Reply:
The !important tag is needed due to the use of bootstrap